astor/rtrip.py

```diff
@@ -85,6 +85,12 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s):
+    if b'' == u'':
```

Comment:
I didn't know about that switch -- that's an awesome workaround for a stupid problem that should have never existed in the first place, so thanks for that tidbit.

Anyway, I'm not quite ready to drop 2.6, since it's not that difficult, and I'm not even ready to use `six` because the problem's not that hard, and I don't want to explicitly check the version, because this is about behavior, not arbitrary numbers.  How about: `if types(b'') == types('')` ?